source/core_http_client.c

```diff
@@ -2254,6 +2250,9 @@ static HTTPStatus_t findHeaderInResponse( const uint8_t * pBuffer,
                     pField,
                     ( int ) ( *pValueLen ),
                     *pValueLoc ) );
+
```

Comment:
The comment on line 2254 doesn't seem helpful, I would delete it.  This `else` block is doing something important, which is to set `returnStatus` to indicate that the requested header was found in the response.

Reply:
you are right thanks